### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 38

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#214] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255192-281d0bb8-ae06-4971-9f16-8cec77116cfc.png)
![image](https://user-images.githubusercontent.com/69389461/152255200-060b380a-ae42-41a8-944b-2f1b2bd1c078.png)

**Comment by i0nut02** [comment]
1.  26^3 * 10^3 * 3
2. 26^3 * 10^3 * 3 + 26^4 * 10^3 - 26^3 * 10^3
3. 4 * 25^3 * 3 * 9^2  * 3
